lib/index.js

```diff
@@ -29,7 +28,7 @@ function buildConfig(pluginConfig, configDir, { compact, manager, browsers }) {
 		if(require.resolve.paths) {
 			includePaths = require.resolve.paths("something");
 		} else {
-			includePaths = [ path.resolve(configDir, "node_modules") ];
+			includePaths = [ manager.resolvePath("./node_modules") ];
```

Comment:
That seems far less hacky than what we'd originally decided on 👏 .

Having said that, it does break encapsulation a little: I'm not entirely sure whether expecting the config file to reside next to `node_modules` will turn out to be a problem - think `-c config/prod.js`. If it does, we'll have to fix that in both faucet-core and faucet-sass. But let's not get bogged down by hypothetical edge cases.